fairroot/base/sim/FairModule.cxx

```diff
@@ -218,13 +208,11 @@ void FairModule::ProcessNodes(TList* aList)
       from ConstructGeometry() of your detector class. Aborting...";
     }
 
-    if (!svList) {
-        svList = new TRefArray();
-    }
     if (!vList) {
         vList = new FairVolumeList();
     }
 
+    auto& svList = fAllSensitiveVolumes;
```

Comment:
Whats your thinking behind keeping `svList`?

Reply:
I a previous edition of this PR, I had three things in one commit:
1. Turn `svList` into a std container
2. Rename to `fAllSensitiveVolumes`
3. Move to a normal member variable of FairRunSim

I realized that splitting this into two commits (one for 1/2, one for 3) would be better.

So caching `fAllSensitiveVolumes` in a local variable is somewhat a preparation for (3). There it would look like

```cpp
auto& svList = FairRunSim::Instance()->fAllSensitiveVolumes;
```

(I know, we should get rid of `FairRunSim::Instance()`! In some way this is showing us, that these methods are probably either in the wrong class, or actually need a reference to FairRunSim.)

Preparing it here makes (3) much more concise.

Reply:
Just for the reference: Moving to FairRunSim is here: https://github.com/ChristianTackeGSI/FairRoot/commit/5a97d7a9d937d6c40b411e6e2c88c9eb37a5848f

I am unsure, whether I should already add it to this PR?

The current version of this PR likely has issues with pyroot (see befca57f5fdfb6afa420669fcc8c2373ef6289eb). Adding the move to normal member variables (of FairRunSim) would instantly solve that.